src/WebJobs.Script.WebHost/Models/HostStatus.cs

```diff
@@ -63,6 +63,12 @@ public class HostStatus
         [JsonProperty(PropertyName = "processUptime", DefaultValueHandling = DefaultValueHandling.Ignore)]
         public long ProcessUptime { get; set; }
 
+        /// <summary>
+        /// Gets or sets a value indicating whether the filesystem is read-only
+        /// </summary>
+        [JsonProperty(PropertyName = "isFunctionAppEditable", DefaultValueHandling = DefaultValueHandling.Ignore)]
+        public bool IsFunctionAppEditable { get; set; }
```

Comment:
Unit tests?

Reply:
Working on it. I created the PR to kick off Azure Pipelines and it confirmed my suspicion that this test never gets executed: `HostStatus_AdminLevel_Succeeds` in c:\git\azure-functions-host\test\WebJobs.Script.Tests.Integration\WebHostEndToEnd\SamplesEndToEndTests_CSharp.cs

Investigating why...

Reply:
Actually, it does get executed in the CI, but it passed. It should fail because I changed the number of properties in HostStatus

Reply:
Found a more suitable place for a new unit test.